language-plutus-core/test/types/addIntegerCorrect.plc

```diff
@@ -1,7 +0,0 @@
-(program 0.1.0
```

Comment:
why is this one gone?

Reply:
They're all gone because they are not normalized.

Reply:
Oh, this is the builtin type value bug again. `[(con integer) (con 2)]` should be a normalized type, but isn't.

Can you move things like this that we expect to work in the future into the type-errors folder with a note, please?

Reply:
> Oh, this is the builtin type value bug again.

Bug in the type synthesis? This conforms to the spec, no?

Reply:
It's a bug in the spec. This is the thing we discussed at the end of the call on the 20th about type level builtins.

Given that it's a bug in the spec and we expect this to work in future, I think its worth keeping things like this around so we can just move them from `type-errors` to `types` when they start working!